internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -109,11 +124,15 @@ param (
     [bool]$insecure = $false,
     [string]$collector_version = "",
     [bool]$with_fluentd = $true,
+    [bool]$with_dotnet_instrumentation = $false,
+    [string]$instrumentation_exclude_processes = "Powershell.exe;dotnet.exe",
```

Comment:
Same as above: 
```suggestion
    [string]$instrumentation_exclude_processes = "Powershell.exe;dotnet.exe;devenv.exe",
```

Reply:
```suggestion
    [string]$instrumentation_exclude_processes = "Powershell.exe;dotnet.exe",devenv.exe,com.docker.service,com.docker.service.exe"
```
It looks like it can be without `.exe` suffix (can you double-check it @pjanotti on your machine?)

However https://docs.docker.com/desktop/windows/permission-requirements/ mentions the `.exe` suffix 🤷

Reply:
I'm not sure what we try to achieve here. If we don't go with globally enabled instrumentation, this list here is non important.

If we want to store these just in case, then I would add:
Server Manager - ServerManager.exe
IIS Manager - InetMgr.exe

Reply:
> If we don't go with globally enabled instrumentation, this list here is non important.

You are right 😆

Reply:
This has been removed 🙂

Reply:
+1 we are not moving forward with other .NET processes and the ability to exclude processes for the first version, only IIS apps. @pmcollins has removed that option for now based on yesterday's discussion, we will keep the above processes in mind for future iterations.